Sources/MapboxSearch/PublicAPI/HistoryRecord.swift

```diff
@@ -20,10 +20,18 @@ public struct HistoryRecord: IndexableRecord, SearchResult, Codable, Hashable {
     }
     
     /// Unique identifier
-    public private(set) var id: String
```

Comment:
we can mention in changelog that HistoryRecord.id is now customizable. But it now inconsistent with FavoriteRecord.id which is `let`

Reply:
Just a tail from small refactoring. Firstly, I decided to make it `let` but didn't revert all the changes :)